packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -87,582 +97,842 @@ describe('quantic-generated-answer', () => {
-          Expect.displayRephraseButtons(true);
-          Expect.displayRephraseLabel(true);
+        it('should not display the component', () => {
+          Expect.displayGeneratedAnswerCard(false);
         });
+      });
 
-        it(
-          'should display feedback buttons',
-          {
-            retries: 12,
-          },
-          () => {
-            Expect.displayLikeButton(true);
-            Expect.displayDislikeButton(true);
-            Expect.likeButtonIsChecked(false);
-            Expect.dislikeButtonIsChecked(false);
-
-            scope('when liking the generated answer', () => {
-              Actions.likeGeneratedAnswer();
-              Expect.logLikeGeneratedAnswer(streamId);
-              Expect.likeButtonIsChecked(true);
-              Expect.dislikeButtonIsChecked(false);
-            });
-
-            scope('when disliking the generated answer', () => {
-              Actions.dislikeGeneratedAnswer();
-              Expect.logDislikeGeneratedAnswer(streamId);
-              Expect.likeButtonIsChecked(false);
-              Expect.dislikeButtonIsChecked(true);
-              Expect.displayFeedbackModal(true);
-            });
+      analyticsModeTest.forEach((analytics) => {
+        describe(analytics.label, () => {
+          before(() => {
+            analyticsMode = analytics.mode;
+          });
 
-            scope('when closing the feedback modal', () => {
-              Actions.clickFeedbackCancelButton();
-              Expect.displayFeedbackModal(false);
-            });
+          describe('when a stream ID is returned', () => {
+            describe('when a message event is received', () => {
+              const streamId = crypto.randomUUID();
 
-            scope('when selecting a feedback option', () => {
-              Actions.dislikeGeneratedAnswer();
-              Actions.clickFeedbackOption(
-                feedbackOptions.indexOf(irrelevantOption)
-              );
-              Actions.clickFeedbackSubmitButton();
-              Expect.logGeneratedAnswerFeedbackSubmit(streamId, {
-                reason: irrelevantOption,
+              beforeEach(() => {
+                mockSearchWithGeneratedAnswer(streamId, param.useCase);
+                mockStreamResponse(streamId, genQaMessageTypePayload);
+                visitGeneratedAnswer({useCase: param.useCase});
               });
-              Actions.clickFeedbackDoneButton();
-            });
 
-            scope(
-              'when clicking the dislike button after submiting a feedback',
-              () => {
-                Actions.dislikeGeneratedAnswer();
-                Expect.displayFeedbackModal(false);
-              }
-            );
-          }
-        );
-      });
+              it('should log the stream ID in the search event custom data', () => {
+                if (analyticsMode === 'legacy') {
+                  Expect.logStreamIdInAnalytics(streamId, param.useCase);
```

Comment:
ok this is one of the search events no longer sent correct?

Reply:
exactly 👍